components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -757,6 +766,695 @@ impl Interpolate for LengthOrNone {
         TransformList(Some(result))
     }
 
+    /// https://drafts.csswg.org/css-transforms/#Rotate3dDefined
+    fn rotate_to_matrix(x: f32, y: f32, z: f32, a: SpecifiedAngle) -> ComputedMatrix {
+        let rad = a.radians();
+        let sc = (rad / 2.0).sin() * (rad / 2.0).cos();
+        let sq = 1.0 / 2.0 * (1.0 - (rad).cos());
```

Comment:
this gives different results from (rad/ 2.0).sin().pow(2). difference is small (e-16/17) but can give artifacs